HW3/index.js

```diff
@@ -0,0 +1,6 @@
+let userName = prompt("What is your name?");
+let userAge = prompt("How old are you?");
+let userHobbie = prompt("What is your favourite hobbie");
+alert(`Hello, ${userName}, you are ${userAge}, your favourite hobbie - ${userHobbie}`);
+const number = '12345'
+console.log(number[0], number[1], number[2], number[3], number[4])
```

Comment:
AS an option you could create a variable ```number``` with number type and use ```string``` types to see how it works